README.md

```diff
@@ -1,3 +1,22 @@
-# Guidelines
+#GUIDELINES WIKI
+**Aspiring to be a World-Class software developer/engineer?** This is a guide where you will get engaged on the great values,
+best software development practices and awesome opportunities the Andela-community will offer and equip you with as you journey 
+through to your great dream in your career. 
+This platform provides an avenue whereby the Andela Community Sourced Projects will  allow the community members to create and contribute
+to the open source projects facilitated by Andela and follow the Andela way. This guide will provide examples, templates, and instructions 
+on all of the processes involved in successful project creation so that any community member can start their own team. 
+Once a team has followed our team lead guideline, Andela will assign a Mentor to that project to meet regularly with the team lead and 
+help them overcome any obstacles, as well as celebrate any victories. 
+##PURPOSE OF THE PLATFORM
```

Comment:
Great insight there @goizueta  , sorry  have been away for a week, had some family issue to handle. Now am back on-board, I will be looking into the suggestions, thank you.